src/components/chip-set/examples/chip-set-input-type-text.tsx

```diff
@@ -0,0 +1,106 @@
+import { Chip } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+import { ENTER, ENTER_KEY_CODE } from '../../../util/keycodes';
+
+/**
+ * Input chip set with `inputType` of `text`
+ *
+ * There is a slight difference in the way browsers treat `input` field
+ * with `type="text"` and `type="search"`. You can read more about this
+ * difference in [Mozilla's documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search#using_search_inputs),
+ * but the most important difference in this case is activation of the
+ * autocorrection feature on most smart devices.
+ *
+ * When a user makes a spelling mistake while typing in an input field with
+ * `type="text"`, the mistake will be corrected automatically, right after they
+ * press <kbd>Enter</kbd> or <kbd>Space</kbd>. Input fields with `type="search"`
+ * do not auto correct the user's input.
+ *
+ * If you want to use limel-chip-set in a form context, where autocorrection is
+ * a good thing, use `text` as `inputType`. It is important to know that the
+ * chip-set component creates a chip from the autocorrected value, after the
+ * user has pressed the <kbd>Enter</kbd> key and the auto correction has fixed
+ * existing typos! For example, for a question like "Please type five of your
+ * favorite fruits", you would want to avoid misspellings, to collect higher
+ * quality data.
```

Comment:
Bias declaration: I don't use automatic autocorrection, because I prefer to just get a red underline for potential typos, rather than having my phone change stuff automatically, because I just can't stand when it "autocorrects" something away from what I actually meant to something else. Given that, I have a hard time imagining what a user who likes autocorrection might prefer here. But:

I assume that, if you have autocorrection enabled, if you actually meant what you wrote, you can go back and change it again (I guess you can, right? It won't then autocorrect _again_?). If so, what about when it's used in the chip-set? If you write something that gets incorrectly "autocorrected", remove the incorrect chip, and then write the same thing again, will your phone understand not to change it this time, or will it "autocorrect" it again, since you did "other stuff" in between?

If it would "autocorrect" again, having autocorrection enabled on a chip-set would make it very hard to write anything the autocorrection feature doesn't "allow"…

I guess you could write your word, press space to have it autocorrect, then backspace and correct what you wrote, and only _then_ press enter. But how many users are actually going to figure that out?

As I said, I'm clearly biased, and I'm not even going to be subjected to any of this, since I have autocorrection disabled, but I feel like having autocorrection on a chip-set is exactly the kind of thing that could make users hate our app… 😞

Reply:
> you can go back and change it again (I guess you can, right? It won't then autocorrect again?) […] 

It depends on the device, and timing of when you correct it, where you are typing, and some other factors that I don't really figure out. But it will in many cases. 

>  […] what about chipset […]

I think this also depends on device, timing, etc. On iOS when I try, the browser stops auto-correcting (for a while[?] while the `focus` is still in that field[?]) when I remove the wrong chip or word and try again. But the primary problem here is that user must realize first that what's in the "chip" is not what they wrote. For my dyslexic brain for instance, this may take a very long time sometimes. 

Note that chips don't show that red underline that indicates spelling errors.

> But how many users are actually going to figure that out?

If you realize that this autocorrection happens, you can find a way to hack it. But first you should consciously realize it.

Reply:
I agree with your assessment above. It's unlikely the user will notice any autocorrection will happen, and it's even less likely they will figure out how to get around it, in case the autocorrect didn't do what they wanted. So I think we should just disable autocorrect for all limel-chip-sets, regardless of whether the input type is text or search…

I think we can do that either in this PR or separately, but if we want to do that, it seems quite unnecessary to now add a long explanation of why enabling autocorrection is good in some cases. That's the only reason I'm bringing this up here, rather than just merging 🙂

Reply:
yes makes sense. If we want to introduce `auto-correction= off` then we should not explain these stuff for input type. And at least I don't know a if we will have any more legit reasons to keep input type (we're overwriting the browser styles anyways). Or do you think we should keep both as props?

Reply:
We could keep both, but if the main point is to disable autocorrection, it would be a smaller change to just add that in limel-chip-set, without changing the interface at all. Then we can always add the inputType option if we feel an actual need for it.

Reply:
sounds good to me

Reply:
what do you think @LucyChyzhova ?

Reply:
Oh actually when I read [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search#search_form_labels_and_accessibility) again, I change my mind. Autocorrection is one of the things we want to remove, when the user action is about searching. But that doesn't solve the accessibility problems.

So in this search field 
<img width="480" alt="image" src="https://user-images.githubusercontent.com/35954987/169243029-b8abf980-4fd1-4432-8bb4-f3b5eb7fc239.png">
a screen reader wouldn't give enough information to users. Specially because our label ("Begins with") is also confusing. It becomes more clear for a sighted user, when we show the magnifying glass icon there. But not-sighted users won't see that. They would need more clues to realize they can actually perform a search.

Reply:
The way you describe that autocorrection works for limel-chip-set, it _really_ doesn't sound to me like we would want autocorrection there, since it would be pretty much impossible for a user to write something that their phone thinks it ought to autocorrect, in case the user doesn't actually want it "corrected".

But I'll leave that up to you, since you are our interaction designer, after all 😊